### ISSUE TITLE
[Bug] defineAppConfig is not defined

### ISSUE THREAD

**Issue by larrasu** [nuxtlabs/docus#830]
I'm getting the error: `defineAppConfig is not defined`.

To reproduce:
1. Install Docus as is from the docs with `npx nuxi@latest init docs -t themes/docus`
2. Run `yarn install`
3. Got the error

**EDIT:** I tried installing with `npx nuxi init docs -t themes/docus` on an empty parent folder and it worked.

![image](https://user-images.githubusercontent.com/74983120/221363592-06869c23-dd23-4af9-9c7c-bfa8b7e0d4f1.png)

**Comment by larrasu** [comment]
I'm making a new module for Nuxt, but I can't make docs using Docus because of this. It only works if the root folder is empty. Even though the root folder has the same Nuxt version as the docs, it doesn't work.

**Comment by zoey-kaiser** [comment]
> I'm making a new module for Nuxt, but I can't make docs using Docus because of this. It only works if the root folder is empty. Even though the root folder has the same Nuxt version as the docs, it doesn't work.

We have the same error for a while but sadly have not found a solution https://github.com/nuxt-themes/docus/issues/680

**Comment by Tahul** [comment]
@danielroe ; it's not the first time I hear from these kind of issues with auto-imports in root project configuration files.

I've also encountered a `defineNuxtConfig is undefined` in the `nuxt/image` docs directory lately.

Is there something I'm missing here or a suggested workaround in case this happens?

Thanks 🙏

**Comment by zoey-kaiser** [comment]
> Is there a possibility that there is a `.nuxtrc` file in a parent directory that sets `imports.autoImports=false`?

Yes there is! Can we override it with a new `.nuxtrc` in the docs subfolder?

**Comment by danielroe** [comment]
Absolutely, you can. :+1:

Here's an example: https://stackblitz.com/edit/github-jhx2ud

**Comment by 1714080902120** [comment]
I got the same problem, is there any solution?
